rbd/rbd.go

```diff
@@ -49,6 +49,8 @@ const (
 // Timespec is a public type for the internal C 'struct timespec'
 type Timespec ts.Timespec
 
+// revive:disable:var-naming old-yet-exported public api
```

Comment:
I'm not so happy with linter specific instructions. ~Normally you can switch these things more generically with `nolint: ...` comments. Did you try that?~ But, revive doesn't support that, that is sad.

Reply:
Also, I read somewhere that comment directives should not have a leading space, so it should be `//revive:...`

Reply:
Tried that. Didn't work.

Reply:
You mean without space? I tried it here and it did work. 🤔

Reply:
Also the revive docs examples don't have a space.

Reply:
Agreed on the examples..... that's what I had put at first. But it still alerted on the items, but when I added the space it started to work as I expected i to. Maybe I need a newer version.